packages/web-api-client/src/register-web-api-client-library-to-container.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as inversify from 'inversify';
+import { A11yServiceClient } from './a11y-service-client';
+
+export function registerWebApiClientLibraryToContainer(container: inversify.Container): inversify.Container {
```

Comment:
this should not be in this package. let this be inside the web-worker's container setup.